webapi/src/main/twirl/queries/sparql/v1/getVersionHistoryGraphDB.scala.txt

```diff
@@ -72,11 +72,6 @@ WHERE {
         BIND(true as ?isLinkValue)
     }
 
-    OPTIONAL {
-        ?value knora-base:attachedToUser ?valueCreator .
```

Comment:
How could this be optional? Did the owner on a value use to be optional in knora-base?